### ISSUE TITLE
False positive: numpy ndarray .T does not support item assignment

### ISSUE THREAD

**Issue by anuppari** [PyCQA/pylint#3932]
## Code to reproduce:
```python
import numpy as np

a = np.array([[1,2,3],[4,5,6]])
print(a)

a.T[1,1] = 10
print(a)
```

## Current behavior
Pylint gives
> E1137: 'a.T' does not support item assignment (unsupported-assignment-operation)

but the program runs fine

## Expected behavior
No pylint error.

## pylint --version output
pylint 2.6.0
astroid 2.4.2
Python 3.8.5 (default, Aug  6 2020, 16:46:05) 
[GCC 7.5.0]

**Comment by hippo91** [comment]
@anuppari thanks for the report. It is the same issue as #3387 so i close this one.
